### ISSUE TITLE
An unhandled exception occurred: context.getProjectMetadata is not a function

### ISSUE THREAD

**Issue by koczka** [ionic-team/angular-toolkit#215]
I get this error message, when I try to run my ionic app in the simulator.

An unhandled exception occurred: context.getProjectMetadata is not a function

ionic cordova run ios --livereload --target 'C0DE8C80-1028-4416-A2D5-06D81A4F543A'

My Ionic info
`Ionic:

   Ionic CLI                     : 6.1.0 (/Users/user/.nvm/versions/node/v10.19.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.0.1
   @angular-devkit/build-angular : 0.900.3
   @angular-devkit/schematics    : 9.0.3
   @angular/cli                  : 9.0.3
   @ionic/angular-toolkit        : 2.2.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-webview 4.1.3, (and 14 other plugins)

Utility:

   cordova-res : not installed
   native-run  : 0.3.0

System:

   ios-deploy : 1.10.0
   ios-sim    : 8.0.2
   NodeJS     : v10.19.0 (/Users/user/.nvm/versions/node/v10.19.0/bin/node)
   npm        : 6.13.7
   OS         : macOS Catalina
   Xcode      : Xcode 11.3.1 Build version 11C504 `

**Comment by dopplerDistortion** [comment]
Experienced the same issue today. Check your angular package versions in your package.json. For Example mine was like this
`"dependencies": {
		"@angular/animations": "^9.0.2",
		"@angular/common": "^9.0.2",
		"@angular/compiler": "^9.0.2",
		"@angular/core": "^9.0.2",
		"@angular/forms": "^9.0.2",
		"@angular/platform-browser": "^9.0.2",
		"@angular/platform-browser-dynamic": "^9.0.2",
		"@angular/router": "^9.0.2",
                 ...
},
"devDependencies": {
		"@angular-devkit/architect": "0.803.14",
		"@angular-devkit/build-angular": "0.900.3",
		"@angular-devkit/core": "9.0.3",
		"@angular-devkit/schematics": "9.0.3",
		"@angular/cli": "9.0.2",
		"@angular/compiler-cli": "9.0.2",
		"@angular/language-service": "9.0.2",
`

Then I changed them like this after that my problem is solved

`"dependencies": {
		"@angular/animations": "^9.0.2",
		"@angular/common": "^9.0.2",
		"@angular/compiler": "^9.0.2",
		"@angular/core": "^9.0.2",
		"@angular/forms": "^9.0.2",
		"@angular/platform-browser": "^9.0.2",
		"@angular/platform-browser-dynamic": "^9.0.2",
		"@angular/router": "^9.0.2",
                 ...
},
"devDependencies": {
		"@angular-devkit/architect": "0.900.2",
		"@angular-devkit/build-angular": "0.900.2",
		"@angular-devkit/core": "9.0.2",
		"@angular-devkit/schematics": "9.0.2",
		"@angular/cli": "9.0.2",
		"@angular/compiler-cli": "9.0.2",
		"@angular/language-service": "9.0.2",
`

At first I tried variants of `9.0.3` and `0.900.3` but I ecnounted a flattenedMap error. So I changed it to `9.0.2` and `0.900.2` variants. fyi

**Comment by koczka** [comment]
Yeah, that problem gone away, but I still cannot load the project properly, I always get "Cannot Get /" error now

**Comment by dopplerDistortion** [comment]
> Yeah, that problem gone away, but I still cannot load the project properly, I always get "Cannot Get /" error now

Please provide more detailed logs so we can help you.

**Comment by prjcontrol** [comment]
This solved it for me:
npm install @angular-devkit/architect@latest
npm install @angular-devkit/build-angular@latest

**Comment by pimol** [comment]
The problem appears when:
1) I updated to ionic  5.1.1 from 5.0.7: the app still run
2) I run `npm audit fix` and the error start appearing

For  me, it was fixed [upgrading to angular 9 ](https://update.angular.io/#8.0:9.0)

**Comment by gleysonlf** [comment]
Thank you @prjcontrol!

This solved it for me:
npm install @angular-devkit/architect@latest
npm install @angular-devkit/build-angular@latest

**Comment by sumitdeyonline** [comment]
This solved it for me  as well
